### ISSUE TITLE
Sons of Hodir ase camp dissapearing

### ISSUE THREAD

**Issue by metalinvader989** [dalaranwow/dalaran-wow#3677]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

**Description**:
Sons of Hodir base camp returning to pre quest line state
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After the quest line of sons of Hodir, you can only visit their base camp once. When you leave and come back, Dun Niffelem appears as it was before the quest line was completed.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Do the They Took Our Men! quest chain
2. Unlock the sons of hodir base camp 
3. Leave the area and return

**Include proofs for this behaviour**
In the included screenshot, you can see that I have a quest from the Sons of Hodir and yet I am seeing the pre quest version of the area
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Dun Niffelem
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_082416_180203](https://cloud.githubusercontent.com/assets/21222593/17935733/084f9bc2-6a25-11e6-8cea-a5101f3b8b47.jpg)

**Comment by Golganeth** [comment]
Can't reproduce. Make sure you have completed all of the required quests, because if you're missing just one, this could have caused to revert the phasing.
Several addons can help you track if you are missing any quest.

If the issue persists, please make an in-game ticket and we'll check further into it.

**Comment by metalinvader989** [comment]
Greetings dear gm

In the screenshot I provided it is clearly shown that I already have a quest of the faction of Hodir Sons. Yet the area appears as it was before the quest chain which should be impossible as I have taken a sons of Hodir quest. At the moment I am waiting on an in game ticket. Thanks for the fast response though.

Best regards.
Λήψη του Outlook για Android<https://aka.ms/ghei36>



On Fri, Aug 26, 2016 at 7:59 PM +0300, "Golganeth" <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #3677<https://github.com/dalaranwow/dalaran-wow/issues/3677>.

-

**Comment by kalvindeane** [comment]
Go to Temple of Storms and speak to Thorim.

If he has a quest available, that's the reason why you are in a different phase than the one you are expecting.
